app/views/calls/_new_call.html.erb

```diff
@@ -6,6 +6,15 @@
           <button type="button" class="close close-terms" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
           <h4 class="modal-title calls-request">Call <b><%= p.patient.name %></b> now:</h4>
           <br>
+          <% if !p.patient.other_contact.blank? && !p.patient.other_phone.blank? %>
+          <h4 class="modal-title calls-request calls-other-contact">
+            <%= p.patient.other_contact %>
+            <% if !p.patient.other_contact_relationship.blank? %> (<%= p.patient.other_contact_relationship %>) <% end %>
```

Comment:
Oh yeah I'm happy to do that, I did a quick google to see how to check if a variable is blank and this is what it came up with :D